run.py

```diff
@@ -7,13 +7,18 @@
 import atexit
 import socket
 import numpy as np
+
 from multiprocessing import Process
 from flask import Flask, request, jsonify, abort, url_for, send_file
 from flask_script import Manager
+from flask_cors import CORS
+from quickslice.build import build
+from quickslice.slice import slice
```

Comment:
Don't pull `slice` into the global namespace, it is a reserved Python keyword. Just import `quickslice` and access the names `build` and `slice` using the dot.